src/util/psbt/error.rs

```diff
@@ -65,6 +86,11 @@ impl fmt::Display for Error {
                 f.write_str("partially signed transactions must have an unsigned transaction")
             }
             Error::NoMorePairs => f.write_str("no more key-value pairs for this psbt map"),
+            Error::HashParseError(e) => write!(f, "Hash Parse Error: {}", e),
+            Error::InvalidPreimageHashPair{ref preimage, ref hash, ref hash_type} => {
+                // directly using debug forms of psbthash enums
+                write!(f, "Preimage {:?} does not match hash {:?} :{:?}", preimage, hash_type, hash )
```

Comment:
typo with space at the end of the formatting string:

```suggestion
                write!(f, "Preimage {:?} does not match {:?} hash {:?}", preimage, hash_type, hash )
```